src/org/zaproxy/zap/extension/ascanrulesBeta/ZapAddOn.xml

```diff
@@ -6,6 +6,10 @@
 	<url/>
 	<changes>
 	<![CDATA[
+	Minor code changes.<br>
+	Change scanners to honour the technologies enabled (Issue 1618).<br>
+	Added Buffer Overflow scanner to beta (Issue 1605).<br>
```

Comment:
It looks like I'm gonna have to wipe my local and re pull from the
mrader/zap-extensions storage.

On Fri, Aug 28, 2015 at 5:24 AM, thc202 <notifications@github.com> wrote:

> In src/org/zaproxy/zap/extension/ascanrulesBeta/ZapAddOn.xml
> <https://github.com/zaproxy/zap-extensions/pull/27#discussion_r38187654>:
>
> > @@ -6,6 +6,10 @@
> >  	<url/>
> >  	<changes>
> >  	<![CDATA[
> > +	Minor code changes.<br>
> > +	Change scanners to honour the technologies enabled (Issue 1618).<br>
> > +	Added Buffer Overflow scanner to beta (Issue 1605).<br>
>
> P.S. those should be removed when dealing with rebase conflict though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zaproxy/zap-extensions/pull/27/files#r38187654>.
>